tests/jerry/es.next/regression-test-issue-4129.js

```diff
@@ -0,0 +1,15 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+JSON.stringify("\uD834")
```

Comment:
Please add some assertion here.

Reply:
The goal of this test is to check we don't get heap-buffer-overflow with ASAN. 
But of course, we can check the result of the JSON.stringify() too, which was already correct.